packages/mattermost-redux/src/reducers/entities/integrations.ts

```diff
@@ -195,6 +195,66 @@ function oauthApps(state: IDMappedObjects<OAuthApp> = {}, action: GenericAction)
     }
 }
 
+function appsOAuthAppsIDs(state: string[] = [], action: GenericAction) {
```

Comment:
nit: curious why using `App**s**IDs` (plural) vs `BotIDs` (singular), since they are both apps-related artifacts.

Reply:
There are two main reasons for this:
- Me not being native speaker, and doubting which of the two ways sounded better.
- Me not going over the changes and ensure consistency.

That being said, I have no preference over singular or plural, so which one do you think is better? I will make sure it is consistent for the choice we decide.

Reply:
0/5 on singular vs. plural (prob. singular makes more sense), I just advocate aligning on one vs. another for both.